versions.props

```diff
@@ -24,9 +24,10 @@ com.lmax:disruptor = 3.4.2
 com.netflix.feign:* = 8.18.0
 com.palantir.common:streams = 1.9.1
 com.palantir.config.crypto:encrypted-config-value-module = 2.1.0
-com.palantir.conjure:conjure = 4.7.0
+com.palantir.conjure:conjure = 4.10.1
```

Comment:
Changes here are just bumps, and explicit dependency on dialogue; @ferozco will excavator keep bumping the dialogue version or do we need to do something special?

Reply:
Dialogue will be automatically upgraded by excavator :)